crates/ekore/src/harmonics/cache.rs

```diff
@@ -0,0 +1,48 @@
+//! Cache harmonic sums for given Mellin N.
+
+use num::complex::Complex;
+use std::collections::HashMap;
```

Comment:
In case you care about the performances of this object, some (i.e. a relevant amount of) people are using this library instead of standard library
https://crates.io/crates/hashbrown

Reply:
1fa54f3c